package.json

```diff
@@ -20,10 +20,11 @@
 	"bugs": {
 		"url": "https://github.com/Q42/FloatingFocus/issues"
 	},
-	"main": "dist/floating-focus.js",
+	"main": "dist/styled/floating-focus.js",
```

Comment:
I kinda like this foldered output. (And right ow it's broken without the output. 😏 )

Reply:
pff did I forget removing this one? I'm also becoming less sharp haha

Reply:
Okay but I'll take another shot at doing it in the foldered way (: